Kentico.Kontent.Delivery/ContentItems/PropertyNameAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Kentico.Kontent.Delivery.ContentItems
```

Comment:
What about move of this whole class to namespace (and appropriate folder) `Kentico.Kontent.Delivery.ContentItems.Attributes` ? The benefit would be better structure.

I am thinking about this, because for instance some custom attributes for RichText are already in separate namespace `Kentico.Kontent.Delivery.ContentItems.RichText.Attributes` (and not in root of `...RichText`).

Reply:
Thanks for changing this.